### DISCUSSION TITLE
Overlay Heatmaps over Multiple Maps

### DISCUSSION THREAD

**Discussion by jmacd867** [akiver/cs-demo-manager#878] [Q&A] [answered]
Howdy! I'm antistratting against another team and am currently just exporting pngs of heatmaps, setting the transparency to 50% and overlaying them but I was wondering if a feature could be added where if multiple demos all under the same map could be selected and analyzed to show most common positions players would be based off the side they are on and specific team (not all 10 people)

**Comment by akiver** [comment, marked as answer]
Since v3.6.0 you can generate heatmaps of a team for a given map, example:

![heatmap](https://github.com/akiver/cs-demo-manager/assets/1842524/346c693a-3b24-4d85-8d6b-360eb125ee90)

**Reply by jmacd867** [reply]
ah sorry about that I must have missed that. Is there also a heatmap for pathing? Based off player
